src/org/labkey/test/tests/UserEditorWithoutDeleteTest.java

```diff
@@ -0,0 +1,86 @@
+package org.labkey.test.tests;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.TestFileUtils;
+import org.labkey.test.categories.Daily;
+import org.labkey.test.components.FilesWebPart;
+import org.labkey.test.util.FileBrowserHelper;
+import org.labkey.test.util.PortalHelper;
+import org.labkey.test.util.TestUser;
+import org.labkey.test.util.core.webdav.WebDavUploadHelper;
+import org.labkey.test.util.core.webdav.WebDavUrlFactory;
+import org.labkey.test.util.core.webdav.WebDavUtils;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+
+@Category({Daily.class})
+public class UserEditorWithoutDeleteTest extends BaseWebDriverTest
+{
+    private static final TestUser EDITOR_WITHOUT_DELETE = new TestUser("editorwithoutdelete@user.test");
+    private static final File TEST_FILE = TestFileUtils.getSampleData("studies/Read_Me.txt");
+
+    @BeforeClass
+    public static void setupProject()
+    {
+        UserEditorWithoutDeleteTest init = (UserEditorWithoutDeleteTest) getCurrentTest();
```

Comment:
This cast should no longer be necessary.
```suggestion
        UserEditorWithoutDeleteTest init = getCurrentTest();
```
Do you have warnings disabled in your IntelliJ? It should flag this.

Reply:
I never got any warning, so I think the warning is disabled.

And, this statement was added as part of template.

Reply:
Your branch is a bit behind develop. It's missing the change that makes the cast unnecessary. It is more recent that I remembered (https://github.com/LabKey/testAutomation/pull/2156)